recipe/meta.yaml

```diff
@@ -12,18 +12,19 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 1001
+  number: 1000
   script:
     - export DISABLE_NUMCODECS_AVX2=""  # [unix]
     - set DISABLE_NUMCODECS_AVX2=""  # [win]
-    - python setup.py install --single-version-externally-managed --record record.txt
+    - {{ PYTHON }} -m pip install . -vvv
```

Comment:
Forgot that we still need `--no-deps`. PR ( https://github.com/conda-forge/numcodecs-feedstock/pull/32 ) fixes this. Also have pulled the affected packages.